recipes/itsxpress/meta.yaml

```diff
@@ -13,10 +13,12 @@ source:
   '{{ hash_type }}': '{{ hash_value }}'
 
 build:
-  number: 0
+  number: 1
   entry_points:
     - itsxpress=itsxpress.main:main
-  script: python -m pip install --no-deps --ignore-installed .
+  script:
+    - python -m pip install --no-deps --ignore-installed .
+    - python -m pip install q2-itsxpress
```

Comment:
This should be a real package. Please see here: https://github.com/bioconda/bioconda-recipes/issues/9930

Reply:
Sorry, @bgruening  I don't quite know what you mean.  Q2-itsxpress is a QIIME2 plugin that makes the functionality of itsxpress available in QIIME2. Some plugins are in the core QIIME2 software distribution but mine in not. I had initially tried to create another conda recipe to install q2-itsxpress but it requires QIIME2 as a depedency to run so it can't be installed by bioconda.  I started a discussion on the QIIME forum about this. https://forum.qiime2.org/t/conda-repositories-for-contributed-plugin-packages/4956. 

Then I realized that the plugin could be bundled with itsxpress so when people install itsxpress within a conda environment containing QIIME2 itsxpress is available from within QIIME as well.

Do you have advice on the best way to distribute q2-itsxpress?